client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -21,191 +21,219 @@ import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
-												type="number" />
-									}
-								}
-							) }
-							<FormSettingExplanation>
-								{ translate( 'Listings will display for the set number of days, then expire. ' +
-									'Leave this field blank if you don\'t want listings to have an expiration date.' ) }
-							</FormSettingExplanation>
-						</FormFieldset>
-					</Card>
-				</FormSection>
-			</form>
-
-			<form>
-				<FormSection name="method">
-					<SectionHeader label={ translate( 'Application Method' ) }>
-						<FormButton compact
-							disabled={ isDisabled } />
-					</SectionHeader>
-					<Card>
-						<FormFieldset>
-							<FormSettingExplanation>
-								{ translate( 'Choose the application method job listers will need to provide. ' +
-									'Specify URL or email address only, or allow listers to choose which they prefer.' ) }
-							</FormSettingExplanation>
-							<FormLabel>
-								<ReduxFormRadio
+								<ReduxFormToggle
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="" />
-								<span>
-									{ translate( 'Email address or website URL' ) }
-								</span>
-							</FormLabel>
-
-							<FormLabel>
-								<ReduxFormRadio
+									name="enableRegistration"
+									text={ translate( 'Enable account creation during submission' ) } />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Includes account creation on the listing submission form, to allow ' +
+										'non-registered users to create an account and submit a job listing simultaneously.' ) }
+								</FormSettingExplanation>
+
+								<ReduxFormToggle
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="email" />
-								<span>
-									{ translate( 'Email addresses only' ) }
-								</span>
-							</FormLabel>
-
-							<FormLabel>
-								<ReduxFormRadio
+									name="generateUsername"
+									text="Generate usernames from email addresses" />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+										'email address. If this is not enabled, a "username" field will display instead.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							{ enableRegistration &&
+								<FormFieldset>
+									<FormLabel>
+										{ translate( 'Role' ) }
+									</FormLabel>
+									<ReduxFormSelect
+										disabled={ isDisabled }
+										name="role">
+										<option value="editor">{ translate( 'Editor' ) }</option>
+										<option value="author">{ translate( 'Author' ) }</option>
+										<option value="contributor">{ translate( 'Contributor' ) }</option>
+										<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+										<option value="teacher">{ translate( 'Teacher' ) }</option>
+										<option value="employer">{ translate( 'Employer' ) }</option>
+										<option value="customer">{ translate( 'Customer' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Customer Name' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).